aws/modules/deploy/helms/01-gpii-preferences/README.md

```diff
@@ -1,59 +0,0 @@
-# GPII Preferences
```

Comment:
I understand the renames but I don't understand why some of these files were deleted?

Reply:
There were 2 versions of preferences and flowmanager charts for AWS and GCP parts. I made them universal, so one of the versions is now removed.